tests/eko/test_couplings.py

```diff
@@ -131,244 +131,42 @@ def test_exact_LO(self):
-            )
-        )
-        for thresh_setup in thresh_setups:
-            threshs = MatchingScales.from_dict(dict(zip("cbt", thresh_setup)))
-            # in LO expanded  = exact
-            sc_expanded = Couplings(
-                couplings,
-                (3, 2),
-                method=CouplingEvolutionMethod.EXPANDED,
-                masses=masses,
-                thresholds_ratios=threshs,
-            )
-            sc_exact = Couplings(
-                couplings,
-                (3, 2),
-                method=CouplingEvolutionMethod.EXACT,
-                masses=masses,
-                thresholds_ratios=threshs,
-            )
-            for q2 in [1e1, 1e2, 1e3, 1e4]:
-                np.testing.assert_allclose(
-                    sc_expanded.a(q2)[0], sc_exact.a(q2)[0], atol=5e-4
-                )
-                np.testing.assert_allclose(
-                    sc_expanded.a(q2)[1], sc_exact.a(q2)[1], atol=5e-4
-                )
-
-    def test_exact_N3LO_mix(self):
-        # prepare
-        thresh_setups = [
-            (np.inf, np.inf, np.inf),
-            (0, np.inf, np.inf),
-            (2, 4, 175),
-        ]
-        alpharef = np.array([0.118, 0.00781])
-        muref = 91.0
-        couplings = CouplingsRef.from_dict(
-            dict(
-                alphas=[alpharef[0], muref],
-                alphaem=[alpharef[1], nan],
-                num_flavs_ref=None,
-                num_flavs_max_as=6,
-            )
-        )
-        for thresh_setup in thresh_setups:
-            threshs = MatchingScales.from_dict(dict(zip("cbt", thresh_setup)))
-            # in LO expanded  = exact
-            sc_expanded = Couplings(
-                couplings,
-                (4, 2),
-                method=CouplingEvolutionMethod.EXPANDED,
-                masses=masses,
-                thresholds_ratios=threshs,
-            )
-            sc_exact = Couplings(
-                couplings,
-                (4, 2),
-                method=CouplingEvolutionMethod.EXACT,
-                masses=masses,
-                thresholds_ratios=threshs,
-            )
-            for q2 in [1e1, 1e2, 1e3, 1e4]:
-                np.testing.assert_allclose(
-                    sc_expanded.a(q2)[0], sc_exact.a(q2)[0], atol=5e-3
-                )
-                np.testing.assert_allclose(
-                    sc_expanded.a(q2)[1], sc_exact.a(q2)[1], atol=5e-4
-                )
+                    for q2 in [1, 1e1, 1e2, 1e3, 1e4]:
+                        # At LO (either QCD or QED LO) the exact and expanded
+                        # solutions coincide, while beyond LO they don't.
+                        # Therefore if the path is too long they start being different.
+                        if q2 in [1, 1e1] and pto not in [(1, 0), (0, 1)]:
+                            continue
+                        np.testing.assert_allclose(
+                            sc_expanded.a(q2)[0], sc_exact.a(q2)[0], rtol=5e-3
```

Comment:
`rtol` should still depend on the order as before else we weaken the test for the good ones ...